worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -320,6 +323,37 @@ public void butcherBrush(Player player, LocalSession session,
         player.printInfo(TranslatableComponent.of("worldedit.brush.butcher.equip", TextComponent.of((int) radius)));
     }
 
+    @Command(
+        name = "image",
+        desc = "Image brush, uses images as input"
+    )
+    @CommandPermissions("worldedit.brush.image")
+    public void imageBrush(Player player, LocalSession session,
+                           @Arg(desc = "The name of the image")
+                               String imageName,
+                           @Arg(desc = "The size of the brush", def = "5")
+                               double radius,
+                           @Arg(desc = "The intensity of the brush", def = "5")
+                               double intensity,
+                           @Arg(desc = "Whether to fill or clear blocks", def = "true")
+                               boolean fillMode,
+                           @Arg(desc = "Whether to flatten blocks to the given height", def = "false")
+                               boolean flatMode) throws WorldEditException {
```

Comment:
These arguments should be `@Switch`es, since they are just boolean values.

I also suggest making `radius` and `intensity` `@ArgFlag`s, but that's a more controversial choice IMO. We do have some other commands that require default arguments to be re-provided right now.

Reply:
As far as I can see, none of the other types use argflags for the radius or similar, so that would hit the consistency a bit with just this one